BRAINSABC/brainseg/EMSegmentationFilter.hxx

```diff
@@ -120,50 +120,51 @@ EMSegmentationFilter<TInputImage, TProbabilityImage>::kNNCore(SampleType *
   const signed int threadsToUse = std::max<signed int>(1, (maxNumThreads * 2.0 - baseMemoryNeeded) / 2.0);
   // i.e. 10 cores = 20GB available - 10 base requireed= 10/2 = 5 threads can run.
   const size_t minGrainSize = numTest / threadsToUse;
-  tbb::parallel_for(tbb::blocked_range<LOOPITERTYPE>(static_cast<LOOPITERTYPE>(0), static_cast<LOOPITERTYPE>(numTest), minGrainSize),
-                    [=, &liklihoodMatrix](const tbb::blocked_range<LOOPITERTYPE> & r) {
-                      // each test case is a query point
-                      MeasurementVectorType queryPoint(numFeatures);
-                      // Now we should find K labels correspondence to K neighbors
-                      vnl_matrix<FloatingPrecision> neighborLabels(K, numClasses);
-                      // Weight vector
-                      vnl_matrix<FloatingPrecision> weights(1, K, 0);
-
-                      // compute the distances
-                      typename TreeType::InstanceIdentifierVectorType neighbors;
-                      std::vector<double>                             distances(K);
-                      for (size_t iTest = r.begin(); iTest < r.end(); ++iTest) ///////
-                      {
-                        for (size_t i = 0; i < numFeatures; ++i)
-                        {
-                          queryPoint[i] = testMatrix(iTest, i);
-                        }
+  tbb::parallel_for(
+    tbb::blocked_range<LOOPITERTYPE>(static_cast<LOOPITERTYPE>(0), static_cast<LOOPITERTYPE>(numTest), minGrainSize),
```

Comment:
Was `tbb::blocked_range<LOOPITERTYPE>(static_cast<LOOPITERTYPE>(0),` put on a new line by clang-format? If not, this new line seems unnecessary.

Reply:
Yes, clang-format moved `tbb::blocked_range<LOOPITERTYPE>(static_cast<LOOPITERTYPE>(0), ` to a new line. I think it was past the line character limit.